apps/dashboard/src/components/workflow-editor/steps/conditions/edit-step-conditions-form.tsx

```diff
@@ -102,6 +102,9 @@ export const EditStepConditionsForm = () => {
     const updateStepData: Partial<StepUpdateDto> = {
       controlValues: { ...step.controls.values, skip },
     };
+    if (typeof skip === 'boolean') {
+      updateStepData.controlValues!.skip = null;
```

Comment:
there is no case where controlValues is undefined? like in after step creation?

Reply:
I agree we can just do !skip

Reply:
@SokratisVidros, the `skip` is a result of the formatting query to `jsonlogic`... doing `!skip` should work too